### ISSUE TITLE
Two suggestions for newbies for installing-arm9loaderhax-page

### ISSUE THREAD

**Issue by The1andONLYdave** [Plailect/Guide#829]
Maybe one suggestions is Dup of  #821, didn't want to re-open.

I'm new to 3ds homebrew and non native english speaker with suggestions to make the guide clearer for new people just using this guide without gbatemp, reddit or similar.
Maybe you want to implement?

https://3ds.guide/installing-arm9loaderhax:
**section iv - step 3:**
add "if you were on current fw before starting with this guide and restored NANDmin.bin earlier, you don't have to update here"
(suggested, because i thought some partitions like ctr weren't restored or something else *needs* a succesfull update-cycle from official nintendo servers because it reads like "doesn't matter if you done step 1 or step 2 but you need to do step 3")
alternative something like "if update displays you're already up-to-date don't worry"

**section vi - step 7:**
add "navigate to /", because maybe other versions are somewhere on sdcard and user don't find it in /cias, so maybe search other folders first before scrolling down on rootfs-view. (because guide mentioned earlier "delete otp.bin folder from sd" and not suggests to format sd before beginning with the guide it's possible the card has more older versions in different locations)

**Comment by Plailect** [comment]
>section iv - step 3:

https://github.com/Plailect/Guide/commit/872865870525ccfed8612d5863853463b717c372

>section vi - step 7:

duplicate of #827
